build/common.targets

```diff
@@ -393,4 +393,14 @@ Condition=" ('%(Extension)' == '.dll' OR '%(Filename)' == 'NuGet.CommandLine.XPl
       Text="Newtonsoft.Json must be version $(NewtonsoftJsonPackageVersion) but resolved %(Reference.NuGetPackageVersion)"
       Condition=" %(Reference.NuGetPackageId) == 'Newtonsoft.Json' AND %(Reference.NuGetPackageVersion) != '$(NewtonsoftJsonPackageVersion)' " />
   </Target>
+
+  <Target Name="GetAllTargetFrameworks">
```

Comment:
It's great that this will make it easier for everyone to make similar changes in the future.  Thanks much.

It's not very discoverable.  In 6 months, I'm unlikely to remember this target.  Should we include a doc (under /docs) on how to bump TFM's and diff changes?

Reply:
We don't really have a doc with instructions on that. I can write up something quick if you think it's valuable.

Reply:
I would.  Otherwise, I don't know how I'd find/remember details in the future.  Need not be lengthy.  Not sure about others.

Reply:
Sounds good. I'll add it in the next commit.